internal/log/log.go

```diff
@@ -106,19 +107,30 @@ func (impl) NewLogger(args args.Config) *Logger {
 		logLevel = DebugLevel
 	}
 
-	logger.Handler = level.New(newBaseHandler(os.Stdout), logLevel)
+	handler, _ := newBaseHandler(os.Stdout)
+	logger.Handler = level.New(handler, logLevel)
 
 	return &logger
 }
 
-func loggerBackend(cfg ConfigProvider, haveJournal bool) func() apexLog.Handler {
+func loggerBackend(cfg ConfigProvider, haveJournal bool) func() (apexLog.Handler, error) {
 	logger := cfg.Logger()
 	if logger == "journald" {
 		return newJournalHandler
 	}
 	if logger == "syslog" {
 		return newSyslogHandler
 	}
+	if strings.HasPrefix(logger, "file:") {
```

Comment:
I don't think the logger should create the file and support a custom path.
The file logger should instead expect to find and be able to write on a file with a default path, e.g. /var/log/exodus-rsync.log 
It may just return an error on the current stdout if it can't find or write into such default file. 

The steps for creating and setting the correct permissions on the default log file is handled by the installer/packaging process, e.g. rpm or Containerfile etc.

Reply:
yes, I don't see a real need for setting a custom path from config for the file logger. It could be just hardcoded to /var/log/exodus-rsync.log. If using an hardcoded path doesn't help you with mocking, then lets keep it as it is. 
The `return nil, err` does already uses the current stdout, so this is OK.